pkgs/network/src/interface.zig

```diff
@@ -195,12 +199,38 @@ pub const ReqRespMethod = enum {
 };
 pub const ReqRespRequest = union(ReqRespMethod) {
     block_by_root: types.BlockByRootRequest,
-    status: void,
+    status: types.Status,
```

Comment:
this sttaus request doesn't send status  or does it?

Reply:
it send, req and resp both this data structure